android/orblibrary/src/main/java/org/orbtv/orblibrary/BrowserView.java

```diff
@@ -239,6 +240,23 @@ private void updateScale() {
         });
     }
 
+    private void updateAppResolution(int[] graphicsConfigIds) {
```

Comment:
I'm having some concern about selecting resolution
Based on current code, the highest supported resolution of the app will be selected.
204 spec mentions it's possible that app is able to support more than one resolutions, but doesn't mention how app can support multiple resolutions and switch between them.